src/main/java/service/RamlDiffService.java

```diff
@@ -67,10 +78,35 @@ public void diff(String later, String older) throws Exception {
    * "src/test/resources/01-bookservice.raml"); }
    */
 
-  public static void main(String[] args) throws Exception {
+/*  public static void main(String[] args) throws Exception {
     new RamlDiffService().diff("src/test/resources/github.raml", "src/test/resources/github-api-v3.raml");
   }
-
+*/
+  
+  public static void main(String[] args) throws Exception {
+    get(RAML_DIFF_SERVICE, (request, response) -> { 
+        return RAML_DIFF_SERVICE_AVAILABLE_TEXT;    
+    });
+    post(FIND_ALL_DIFFERENCES_CONTEXT, (request, response) -> {

```

Comment:
Why is a post and not a get. I can see that you are taking a JSON of some form as the post body. However the POST in REST specification has a specific meaning. Wouldnt it be nice to just treat this as query params ?